api/article/models/article.settings.json

```diff
@@ -21,21 +21,23 @@
       "collection": "file",
       "via": "related",
       "allowedTypes": [
-        "files",
-        "images",
-        "videos"
+        "images"
       ],
       "plugin": "upload",
+      "path" : "arn:aws:s3:eu-central-1:436054152060:accesspoint/schmiede-blog-image-access-point",
```

Comment:
Path should be environment dependent. So, for dev, a different bucket (or no bucket) should be used

Reply:
Has to be done for settings.json for all models